src/js/themes/base.js

```diff
@@ -707,6 +708,40 @@ export const generate = (baseSpacing = 24, scale = 6) => {
     //   extend: undefined,
     //   maxHeight: undefined,
     // },
+    fileInput: {
+      // background: {},
+      border: {
+        // color: undefined,
+        side: 'all',
+        size: 'small',
+        style: 'dashed',
+      },
+      dragOver: {
+        border: {
+          color: 'red',
```

Comment:
My impression from the red border is that I'm doing something that is not allowed. Can we change this default behavior?
Also, we don't have a 'red' color in grommet, so this color will default to the browser red which I don't think we should support by default.

Reply:
Dropped a similar comment somewhere else. What about increased border size instead of changing color?